dependency/actions/compile/bionic.Dockerfile

```diff
@@ -57,17 +57,15 @@ RUN apt-get update && apt-get install -y \
       ruby-dev \
       rubygems \
       snmp-mibs-downloader \
+      software-properties-common \
```

Comment:
I can't comment on the specific line, but bionic compilation here is being run on the jammy base image, and the jammy compilation is being run on the stack image (ideally what we want). Is there an intentional reason for this disparity?

Reply:
Both are now based on paketo stack images